content/sensu-go/5.10/guides/reduce-alert-fatigue.md

```diff
@@ -61,21 +65,142 @@ sensuctl handler update slack
 Follow the prompts to add the `hourly` and `is_incident` filters to the Slack
 handler.
 
+#### Using a Filter Asset
+
+If you're not already familiar with [assets][asset-reference], take a minute or two and read over our [guide to installing plugins with assets][asset-guide]. This will help you understand what an asset is and how they are used in Sensu. 
+
+The first step we'll need to take is to obtain a filter asset that will allow us to replicate the behavior we used when we created the `hourly` filter via `sensuctl`. Let's use the [fatigue check asset][fatigue-check-asset] from the [Bonsai Asset Index][bonsai-io]. You can download the asset directly by running the following:
+
+{{< highlight shell >}}
+curl -s https://bonsai.sensu.io/release_assets/nixwiz/sensu-go-fatigue-check-filter/0.1.3/any/noarch/download -o sensu-fatigue-check-filter-asset.yml
+{{< /highlight >}}
```

Comment:
Cool idea to use curl! Up to you, but you can lose a few steps by using:

```
curl -s https://bonsai.sensu.io/release_assets/nixwiz/sensu-go-fatigue-check-filter/0.1.3/any/noarch/download | sensuctl create
```

If you like, you can apply this pattern throughout the guide for one-step resource creation using `echo '....' | sensuctl create`